app/Http/Controllers/ProdController.php

```diff
@@ -11,16 +11,63 @@ class ProdController extends Controller
     use Common;
 
 
-    public function index() {
+    /*    #1)
+     * Display a listing of the resource.
+     */
+    public function LatestProds()
+    {
+        /* 
+        #SELECT * FROM `products` ORDER BY updated_at DESC limit 3;
+        */
+        #$products = Product::orderBy('updated_at', 'desc')->limit(3)->get();
+        $prods = Product::where('pub', 1)
+        ->orderBy('updated_at', 'DESC')
```

Comment:
you can replace `orderBy('updated_at', 'DESC')` by `latest()`

Reply:
ok 4okran mo5tasara el 2 fe3ln